src/coreclr/nativeaot/System.Private.CoreLib/src/System/GC.NativeAot.cs

```diff
@@ -782,5 +782,10 @@ public static TimeSpan GetTotalPauseDuration()
         {
             return new TimeSpan(RuntimeImports.RhGetTotalPauseDuration());
         }
+
+        public static System.Collections.Generic.IReadOnlyDictionary<string, object> GetConfigurationVariables()
+        {
+            throw new NotImplementedException();
```

Comment:
Does this need a tracking issue? I honestly don't know if NativeAOT supports these any of these settings.

Reply:
NativeAOT has same GC, so it supports the same settings.

Making this work in NativeAOT is about replicating the QCall from comutilnative.cpp in the matching NativeAOT file. `RhGetTotalAllocatedBytesPrecise` is a good pattern to follow.

Reply:
If it is that straight forward, I think it is prudent to do that in this PR.

Reply:
This has been implemented and tested.